README.md

```diff
@@ -725,7 +725,7 @@ All statements above will return false if used with `===`
 
 **Otherwise:** Insecure dependencies can render an application completely vulnerable.
```

Comment:
Maybe spice it up: An attacker can simply detect your web framework and attack with all its known vunelerabilities